horizomer/benchmark/run_phylophlan.sh

```diff
@@ -48,14 +45,11 @@ mkdir -p input
 mkdir -p output
 mkdir -p temp
 ln -s ${phylophlan_install_dir}/data
-ln -s ${input_faa_dir} input/genomes
+ln -s ${input_faa_dir} input/all
 
 # command
-cmd="${phylophlan_install_dir}/phylophlan.py -u input --nproc $threads --c_dat temp"
-if [ "$verbose" == "true" ]
-then
-    echo "Command:"$'\n'"  $cmd"
-fi
+cmd="${phylophlan_install_dir}/phylophlan.py -u all --nproc $threads --c_dat temp"
```

Comment:
This is but a naming preference. Here `all` or `input` is to describe the job. It is totally arbitrary.